g3doc/tutorials/adversarial_keras_cnn_mnist.ipynb

```diff
@@ -77,7 +77,7 @@
         "([Goodfellow et al., 2014](https://arxiv.org/abs/1412.6572)) for image\n",
```

Comment:
<p>Are the code snippets in the Recap section for illustration purpose and not intended to be run in this colab? (I think this way because they are not runnable without installing NSL and importing TF in the Setup section below.)</p><p> </p><p>If so, could you change the code blocks into text blocks so that readers won't accidentally run them and encounter errors?</p><p> </p><p>You may use three backticks (<code></code>`) to create [fenced code blocks](https://docs.github.com/en/github/writing-on-github/creating-and-highlighting-code-blocks#fenced-code-blocks) to preserve the code-like appearance.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/neural-structured-learning/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>